src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php

```diff
@@ -54,8 +54,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
             ->end()
             ->children()
-                ->variableNode('product_image_prefix')
-                    ->defaultValue('media/image')
```

Comment:
Why did you remove this parameter? It seems unrelated, and there's no single word about it in the upgrade file.

Reply:
Added in #16037